openkfw/TruBudget
# 862 remote fingerprint

### Checklist




### Description


### How to test

1. Log in as mstein
2. Start creating or editing workflow item
3. Add multiple links and also upload files to create hashes
4. Click on pin icon to display list of files and links
5. Validate files and check if links work




Closes #1862
